### PR TITLE
create README.md

### PR DESCRIPTION
**Pull request by ShurionUA**

### CODE REVIEW

**Review by bilobrovets — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Taxi service
+#### Current web app designed to simulate taxi service of storage data, such as drivers, cars, manufacturers of cars. Using this app you can:
```
Comment (Nick97-git) [inline README.md]:
```suggestion
#### Current web app designed to simulate taxi service. Using this app you can:
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+-	Register new drivers, including login and password to access DB;
```
Comment (Nick97-git) [inline README.md]:
```suggestion
-	Register new drivers
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+-	Delete cars, drivers, manufacturers (access on the “Display All Items” pages)
+-	Assigning an existing driver to one or more cars
+-	Get all information about drivers or cars, or manufacturers.
+-	Get a list of cars assigned to specific driver
+#### Allows access only to authorized users, and also provides the ability to register new one. This project follows the rules of SOLID and was created on 3-layered architecture:
+-	Data access layer(DAO)
+-	Application layer(services)
+-	Presentation layer(controllers)
+### DB schema of tables:
+![diagram_db](http://joxi.ru/ZrJ3VRdhbGbBpm.jpg)
```
Comment (Nick97-git) [inline README.md]: image is cropped

File: README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+### Using Technologies:
+-	Apache Tomcat (v9.0.56)
+-	MySQL(v8.0.22)
+-	JDBC
+-	Javax.servlet(v4.0.1)
+-	JSP
+-	JSTL(v1.2)
+-	Apache log4j(v2.14.1)
+-	Apache Maven(v3.3.2)
+-	Apache Maven Checkstyle Plugin(v3.1.1)
```
Comment (Nick97-git) [inline README.md]: remove this line

File: README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+![diagram_db](http://joxi.ru/ZrJ3VRdhbGbBpm.jpg)
+### Using Technologies:
+-	Apache Tomcat (v9.0.56)
+-	MySQL(v8.0.22)
+-	JDBC
+-	Javax.servlet(v4.0.1)
+-	JSP
+-	JSTL(v1.2)
+-	Apache log4j(v2.14.1)
+-	Apache Maven(v3.3.2)
```
Comment (Nick97-git) [inline README.md]: remove word Apache, it's redundant

File: README.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+-	Apache Maven Checkstyle Plugin(v3.1.1)
+-	HTML
+### Steps for setup:
+- Configure Apache Tomcat for your IDE
+- Install MySQL, MySQL Workbench
+- Launch MySQL Workbench and run script from file “resources/init_db.sql”
+- In “src/main/java/taxi/util/ConnectionUtil” change URL, USERNAME, PASSWORD and JDBC DRIVER with your data
+- In “src/main/resources/log4j2.xml” change path to log file <File name="LogToFile" fileName="_______">
+- After starting tomcat go to http://localhost:your_port that you specified while configuring tomcat.
```
Comment (Nick97-git) [inline README.md]: make a numbered list of it

File: README.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+-	Apache log4j(v2.14.1)
+-	Apache Maven(v3.3.2)
+-	Apache Maven Checkstyle Plugin(v3.1.1)
+-	HTML
+### Steps for setup:
+- Configure Apache Tomcat for your IDE
+- Install MySQL, MySQL Workbench
+- Launch MySQL Workbench and run script from file “resources/init_db.sql”
+- In “src/main/java/taxi/util/ConnectionUtil” change URL, USERNAME, PASSWORD and JDBC DRIVER with your data
+- In “src/main/resources/log4j2.xml” change path to log file <File name="LogToFile" fileName="_______">
```
Comment (Nick97-git) [inline README.md]:
```suggestion
- Change URL, USERNAME, PASSWORD and JDBC DRIVER on your data in “src/main/java/taxi/util/ConnectionUtil” 
- Change path to log file <File name="LogToFile" fileName="_______"> in “src/main/resources/log4j2.xml” 
```

**Review by Nick97-git — CHANGES_REQUESTED** [review]
- DeleteDriverController, DeleteManufacturerController -> remove /all from /drivers/all, /manufacturers/all url patterns
- GetMyCurrentCarsController -> let's move this class in `car` package
- AuthenticationServiceImpl -> remove System.out
- web.xml -> /drivers/mycars -> /drivers/cars

**Comment by Nick97-git** [comment]
image is still cropped. Also I don't see changes according to my comments about your code
Screenshot 2021-12-16 at 10 09 30
d

**Review by Nick97-git — APPROVED** [review]

**Pull request merged**